device/common/profiles/th/gen/BALANCED/buffers.json.j2

```diff
@@ -0,0 +1,2 @@
+{%- set default_topo = 't1' %}
```

Comment:
this is different for each hwsku. for eg, for 7060-D48C8 it is 't0' whereas for 7060 C32 it is 't1'